crime-applications-adaptor/src/test/resources/data/criminalapplicationsdatastore/MaatApplicationNoHomeAddress_toBeMapped.json

```diff
@@ -5,6 +5,7 @@
   "application_type": "initial",
   "submitted_at": "2022-10-24T09:50:04.019Z",
   "date_stamp": "2022-10-24T09:50:04.019Z",
+  "declaration_signed_at": "2022-10-24T09:50:04.019Z",
```

Comment:
This timestamp has the UTC 'Z' at the end whereas the dateOfSignature does not - does that matter?

Reply:
That is a good question - I had just copy-pasted the data however now that you mention it I will double check as we have a potential timezone issue in MAAT!

Reply:
In this case it doesn't make a difference - the serialisation of the `LocalDateTime` always defaults to UTC unless there is an explicit offset added.